Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py

```diff
@@ -413,6 +529,111 @@ def upload_pcap_command():
     })
 
 
+def list_metadata_request(time_frame=None, start_time=None, end_time=None, client_ip=None, server_ip=None,
```

Comment:
move those function below the `upload_pcap` function (to keep relative functions together)